code-challenges.test.js

```diff
@@ -38,21 +79,68 @@ const num3 = -7
 
 // b) Create the function that makes the test pass.
 
+//Function will take in a number as an argument and decide if the number is evenly divisible by 3, or not. 
+//Function name: divisibleBy3
+// Input: number
+const divisibleBy3 = (number) => {
+
+    // Use conditional statements with modulo to calculate division, followed by return line with expected output
+    if(number % 3 === 0){
+         return `${number} is divisible by three`
+  } else{
+         return `${number} is not divisible by three`
+  }
+}
```

Comment:
Wonderful!  Since your function is being invoked from the test, there is no longer any need to invoke the function elsewhere like on lines 94 through 96.